### ISSUE TITLE
Client Secret with umlaut is not encoded

### ISSUE THREAD

**Issue by WolfgangHG** [swagger-api/swagger-ui#10180]
I defined a client secret with umlauts in my web service (using OpenIddict for authentication in the service).
When authenticating through SwaggerUI, the login is rejected:
![swaggerui](https://github.com/user-attachments/assets/88db7bc0-a113-4639-b199-d0e02ee2ea00)

Using Restfox, it works without problems. And Restfox gave me a hint: the request body is url encoded:

`client_id=myservice&client_secret=%C3%B6rotkg%C3%B6odrtbjpo%C3%B6id%C3%B6woi4mfwp4wf3tgwp94w9&grant_type=client_credentials`

So, I entered the client secret in SwaggerUI the same way, and it worked!

Thus I assume that SwaggerUI does not encode the client secret when sending the request, and I think it is a bug.

This is my original client secret:
`örotkgöodrtbjpoöidöwoi4mfwp4wf3tgwp94w9`
And this is url encoded:
`%C3%B6rotkg%C3%B6odrtbjpo%C3%B6id%C3%B6woi4mfwp4wf3tgwp94w9`

**Comment by robert-hebel-sb** [comment]
We are doing some housekeeping on our issue tracker. This issue has been inactive for over a year and we have not been able to tackle it yet.

Is this still important to you? If so, please leave a comment – we'd love to know if you're still affected. If we don't hear back in 7 days, we'll close this issue to tidy up the tracker.

Thank you for your understanding!

**Comment by WolfgangHG** [comment]
Yes, it still happens with recent Swashbuckle 10.2.3, which bundles SwaggerUI 5.32.7
